docs/pages/database-access/guides/oracle-self-hosted.mdx

```diff
@@ -188,7 +188,79 @@ NAME                    READY   STATUS    RESTARTS   AGE
 teleport-kube-agent-0   1/1     Running   0          32s
 ```
 
-## Step 5/5. Connect
+## Step 5/6. Configure Teleport Oracle Audit Logs Puller (Optional).
+
+Teleport can pull audit logs from Oracle Audit Trail.
+In order to enable this feature, you will need to configure Oracle Audit Trail and create a dedicated Teleport
+user that will be used to fetch audit events from Oracle Audit Trail.
+
+Create a internal Oracle `teleport` user that will be used as db user that will be used to fetch
+audit events from Oracle Audit Trail:
+
+```sql
+CREATE USER teleport IDENTIFIED EXTERNALLY AS 'CN=teleport';
+GRANT CREATE SESSION TO teleport;
+GRANT SELECT ON dba_audit_trail TO teleport;
+GRANT SELECT ON V_$SESSION TO teleport;
+```
+
+Enable table Oracle Audit Trail:
+```
+ALTER system SET audit_trail=db,extended scope=spfile;
+```
+
+<Admonition type="info" title="audit_trait">
+In order to propagate audit_trail changes, Oracle instance needs to be restarted.
+</Admonition>
+
+Enable Audit Oracle activity for the `alice` user:
+```sql
+AUDIT ALL BY alice BY ACCESS;
+AUDIT SELECT TABLE, UPDATE TABLE, INSERT TABLE, DELETE TABLE BY alice BY ACCESS;
```

Comment:
If my understanding is correct, there are a few more statements not covered by `ALL` like `ALTER TABLE`, `GRANT PROCEDURE`?

Maybe leave a link in the next steps to learn about AUDIT?

I also wonder if `AUDIT ALL STATEMENTS` would help in this case.